### ISSUE TITLE
Feature Requests (2 related to bugs)

### ISSUE THREAD

**Issue by Spiritwind** [CitizensDev/Citizens#821]
Hey guys,
First off I want to say that this is one of my favorite plugins for Minecraft!  I respect the time and effort you have put into making this!

I have some feature requests - I would be willing to donate to see them done :).

Mercenary Guards - Can create a Mercenary guard that can be hired by other players (Owner gets the money), and it will follow the player that hired the guard until either the player dies or the guard dies.  Player that owns the guard can increase the Mercenaries health through funds along with how much it costs to hire the guard.  Can be set in the citizens.yml to where if the player the Mercenary is assigned to dosent log in for so many days then the guard will revert back to being able to be hired.

God Mode Guards - On my servers I use both factions and Heroes - I use factions safezone for any official buildings/spawn.  
In the safezone I have various guards as it is an RPG server.  The problem is that players can kill these "Official" guards and the guard will drop their Armor and Weapon, give PVP exp by Heroes, and then respawn shortly after with their prior inventory so the cycle can be repeated...  It would be really awesome if you could add a feature to where Guards can be put into God mode so that this dosent happen!  Or perhaps even a "dont drop loot" flag.

Talkclose - It gets really aggravating when NPCs talk when you are two floors above them.  I know you have had this suggestion a lot, but it would be really awesome if you made it so that the radius extends a lot less distance above!  Perhaps like 3 blocks?

That is all I can think of right now, if I get any other ideas that I think you guys might want to hear about then I will post them in comments!

Thanks a ton for taking the time to look at this!

-Spiritwind
-President of Imagicraft.net

**Comment by charlie1990** [comment]
As far as them talking to people far away why not just change their talk radius in citizens.yml? Or are you just not wanting them to be able to talk to people above/below them? Was a little confusing to me.

**Comment by BlueLiquidPlus** [comment]
Mercenary - would be great.

God-Mode Guards - These are a must... even with the dev build of the guard.jar I got that prevents items from dropping from dead guards I'd still rather have them be able to be God guards... plus having DeadmansChest plugin as I've yet to figure out how to stop the guards from dropping deathchests this would be great.

Talk-Close - Yeah I agree... reducing the "TalkClose" height to probably the same Y that you are on would be best... I really hate being on the floor above an NPC that I have set to TalkClose start talking to me. =\

**Comment by Spiritwind** [comment]
I mean exactly what Blueliquidplus commented on for Talk-Close charlie1990.

Thank you guys very much for your feedback on my suggestions :).

**Comment by charlie1990** [comment]
yes I definitely agree with talkclose, our beginner quest area can't be entered again once you finish it but the npc's still talk to people on the floor below it. And while were on the subject of guards I really wish if you give them enchanted equipment that they could use it.

**Comment by mazius123** [comment]
i think i should donate too , this plugin just like add feeling to minecraft ... , and Spiritwind  have nice ideas here ;)
